src/quantum/azext_quantum/vendored_sdks/azure_quantum/quantum_client.py

```diff
@@ -49,7 +49,7 @@ def __init__(
         if workspace_name is None:
             raise ValueError("Parameter 'workspace_name' must not be None.")
         if not base_url:
-            base_url = 'https://app-jobscheduler-prod.azurewebsites.net'
+            base_url = 'https://quantum.azure.com'
```

Comment:
Is `quantum.azure.com`, without a region, a valid URL to make requests to? It just gives me a DNS error when I try to hit it; I don't think anything exists there.

I understand the CLI code will override this with the appropriate URL, but I just wonder whether we should either make this parameter required *or* have some valid default. e.g. should we default to `westus.quantum.azure.com` instead?

Reply:
there is no default URL for the service, as it must always include a region, and there is technically no default region. I think it's fine to point to quantum.azure.com even if it's invalid.

Reply:
In any case, this is only a generated file. The source of this URL is in a separate place, which has an explanatory description. I hope in any case that by the time of GA, we don't need this location dependency anymore.
https://github.com/anpaz/azure-rest-api-specs-pr/pull/1